packages/dd-trace/src/appsec/activation.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Activation = {
+  OneClick: 'OneClick',
+  Enabled: 'Enabled',
+  Disabled: 'Disabled',
```

Comment:
We generally reserve uppercase-first (PascalCase) for class names only. For constants we do all uppercase.